### PR TITLE
feat(connect-common): add more backend urls for solana

### PR DESCRIPTION
**Pull request by PeterBenc**
- adds solana1-4 backend urls

### CODE REVIEW

File: packages/connect-common/files/coins.json [hunk 1]
```diff
@@ -2201,7 +2201,13 @@
         {
             "blockchain_link": {
                 "type": "solana",
-                "url": ["https://solana.trezor.io"]
+                "url": [
+                    "https://solana.trezor.io",
```
Comment (mroz22) [inline packages/connect-common/files/coins.json]: lets check with @Dehumanizer77 if we realy run `solana.trezor.io` it does not match pattern of other coins that always have a number suffix

**Comment by komret** [comment]
@PeterBenc @gabrielKerekes Please make these fix PRs against the develop branch if possible, we cherry-pick from it rather than committing directly to the release branch. I merged https://github.com/trezor/trezor-suite/pull/10304 accidentally.

**Comment by PeterBenc** [comment]
@komret rebased on develop

@mroz22 I removed `solana.trezor.io` to be consistent

**Review by vdovhanych — APPROVED** [review]

**Comment by komret** [comment]
@PeterBenc Github checks somehow got stuck, can you please rebase and push again so that they restart? I want to merge this asap.

**Comment by PeterBenc** [comment]
One moment, @Dehumanizer77 is adding `solana5`, I ll add it here and then rerun the checks.

**Comment by PeterBenc** [comment]
Ok, not adding Solana5 for now, reopening to rerun the checks.

**Comment by PeterBenc** [comment]
Ok, reopening did not help, so I rebased. Also checks seem to be waiting for `Expected — Waiting for status to be reported`. Is there some actions you guys need to do make them run?

**Comment by komret** [comment]
It is running now, thanks, I had to approve the workflows. Didn't see it before.

**Comment by komret** [comment]
@trezor/qa Please check the new backends, we will add them in 23.12.2.

**Comment by bosomt** [comment]
**QA OK**


**Info:**
 - Suite version: desktop 23.12.2 (66925d36a98887aac3ab7143bc442ed3bc70a061)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/23.12.2 Chrome/118.0.5993.129 Electron/27.0.4 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T2B1 2.6.4 regular (revision 42e9ed0e09033d474dee1a560fe5870646fa440e)
 - Transport: BridgeTransport 2.0.33

**Pull request merged**